src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java

```diff
@@ -233,4 +233,13 @@ Integer linkDefendant(
         @Bind("defendantId") String defendantId,
         @Bind("defendantEmail") String defendantEmail
     );
+
+    @SqlUpdate(
+            "UPDATE claim SET claim_documents = :claimDocumentCollection::JSONB"
+            + " WHERE id = :claimId"
+    )
+    Integer saveClaimDocuments(
```

Comment:
This method should be updating operation indicators not the claim documents. This looks duplicate of:
https://github.com/hmcts/cmc-claim-store/blob/9024dad6a1c35b240077713377d74af627789108/src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java#L152

Reply:
@sabahirfan this is partial commit. I was working when I have to commit.